### ISSUE TITLE
Interfaces node gets clobbered from skill.json after jovo build with stages

### ISSUE THREAD

**Issue by rmtuckerphx** [jovotech/jovo-cli#43]
## I'm submitting a...

- [X] Bug report
- [ ] Feature request
- [ ] Documentation issue or request
- [ ] Other... Please describe:

## Expected Behavior

Allow specifying the interfaces node in skill.json and have it not be removed when doing a build and using the stages feature.

## Current Behavior

Using the stages feature specifying different endpoints. If my skill.json file has an interfaces node like this:

**app.json**
```json
"stages": {
    "debug": {
      "endpoint": "https://xyz-ABC.bespoken.link",
      "alexaSkill": {
        "skillId": "amzn1.ask.skill.4..."
      }
    },
```

**skill.json (before)**

```json
        "apis": {
            "custom": {
                "endpoint": {
                    "sslCertificateType": "Wildcard",
                    "uri": "https://xyz-ABC.bespoken.link"
                },
                "interfaces": [
                    {
                        "type": "RENDER_TEMPLATE"
                    }
                ]
            }
        },
```

And then I do a build like the following, the interfaces node is clobbered/removed:

`jovo build -p alexaSkill --stage debug`

**skill.json (after)**

```json
        "apis": {
            "custom": {
                "endpoint": {
                    "sslCertificateType": "Wildcard",
                    "uri": "https://xyz-ABC.bespoken.link"
                }
            }
        },
```

### Error log
```
None
```

## Your Environment

* Jovo CLI version used: 1.1.6
* Operating System: Windows 10

**Comment by jankoenig** [comment]
Thank you. We're working on the fix. In the meantime, you can fix this by adding the following to your `app.json`:

```
"alexaSkill": {
        "manifest": {
            "apis": {
                "custom": {
                  "interfaces": [
                    {
                      "type": "RENDER_TEMPLATE"
                    }
                  ]
                }
            }
        }
    },
```

Here is more information about how to override `manifest` elements: https://www.jovo.tech/docs/app-json#alexaskill

**Comment by aswetlow** [comment]
Fixed! Thanks, @rmtuckerphx

**Comment by jankoenig** [comment]
Hi Mark, this got fixed with Jovo CLI 1.1.9 (https://github.com/jovotech/jovo-cli/blob/master/CHANGELOG.md).

Could you update to the latest CLI version and give it another try?

**Comment by aswetlow** [comment]
My bad. It's published now
